fcs/build.fsx

```diff
@@ -21,7 +21,7 @@ let isMono = false
 #endif
 
 
-let dotnetExePath = DotNetCli.InstallDotNetSDK "2.1.200"
+let dotnetExePath = DotNetCli.InstallDotNetSDK "2.1.401"
```

Comment:
@alfonsogarciacaro 2.1.402 is the preferred version as everything below has vulnerability (not applicable here but still).